src/qml/controls/Setting.qml

```diff
@@ -26,6 +26,7 @@ Control {
             }
             OptionSwitch {
                 Layout.alignment: Qt.AlignRight
+                Layout.rightMargin: -12
```

Comment:
It's not clear to me why we are hardcoding a `rightMargin` value of `12` here. This doesn't seem like the correct way to do this, what is the design spec that inspired this change?

Reply:
The `rightMargin` was explicitly set to correctly position the `OptionsSwitch`, to be precisely above the right end of the divider line.

|Before|After|
|---------|------|
|![Screenshot from 2022-08-01 18-07-52](https://user-images.githubusercontent.com/85434418/182149960-f57f0266-7806-4733-88c8-cccb466e82d9.png)|![Screenshot from 2022-08-01 18-08-04](https://user-images.githubusercontent.com/85434418/182149995-bafc6d03-e1be-4a14-baeb-70d68e215e53.png)|

I experimented with other _conventional_ methods of aligning elements using `Layouts` and `Anchors` before going with hardcoded margin. Still, none of them worked to position the Switch over the divider properly.